src/dvbapi.c

```diff
@@ -220,6 +220,7 @@ int dvbapi_reply(sockets *s)
 						k->filter_id[fpos] = -1;
 						k->pid[fpos] = -1;
 					}
+					break;
```

Comment:
Hi @catalinii ,

I'm sure this change is necessary.
It solves problems with my Oscam server.

Reply:
Hi @catalinii ,

You tested this? It _really_ resolves troubles!